src/components/book-card.astro

```diff
@@ -44,11 +46,15 @@ const numberClasses = twMerge(
   style={`--book-color: ${color}`}>
   <div class:list={numberClasses}>
     {
-      collection === "briefs" && (
+      (collection === "briefs" && (
         <span class="text-xs uppercase font-semibold tracking-wider">
           Briefs
         </span>
-      )
+      )) || (collection === "lost" && (
+        <span class="text-xs uppercase font-semibold tracking-wider transform -rotate-45">
+          Lost
+        </span>
+      ))
```

Comment:
open to code golf on this - it doesn't seem TOO bad, and I don't anticipate more categories to emerge so 🤷

Reply:
This seems fine for now, I or a future PR submitter can always refactor if it seems needed. 👍🏻